packages/angular_devkit/build_angular/BUILD.bazel

```diff
@@ -186,6 +186,7 @@ ts_library(
         "@npm//terser-webpack-plugin",
         "@npm//text-table",
         "@npm//tree-kill",
```

Comment:
`@npm//@babel/helper-annotate-as-pure` is missing from here, how is the test even working?

Reply:
Since that package doesn't provide typings, they were added inside `@angular-devkit/build-angular` which is all Bazel needs to compile.

Reply:
But isn’t this needed to run the tests?

Reply:
The magic of package hoisting.  I'll add it here as well though.